lib/main.ts

```diff
@@ -49,10 +49,9 @@ export interface TranspilerOptions {
 }
 
 export const COMPILER_OPTIONS: ts.CompilerOptions = {
-  allowNonTsExtensions: true,
   experimentalDecorators: true,
-  module: ts.ModuleKind.CommonJS,
-  target: ts.ScriptTarget.ES5,
+  module: ts.ModuleKind.ES2015,
```

Comment:
Maybe comment why ES2015 is used rather than say ES2019?

Reply:
Can I just say that it's because it's the most widely supported by browsers? That's the only reason I know haha

Reply:
well this code doesn't run in browsers so that doesn't really matter.
You could just comment that we don't require any features introduced after ES2015 so we have left that as the default version.